Gemfile

```diff
@@ -22,6 +22,8 @@ if ENV['GEM_SOURCE'] =~ /artifactory\.delivery\.puppetlabs\.net/
   gem "scooter", *location_for(ENV['SCOOTER_VERSION'] || '~> 3.0')
 end
 
+gem 'deep_merge'
```

Comment:
This seems like it might be needed in the gem's runtime dependencies in the gemspec.